docs/installation.md

```diff
@@ -28,12 +28,36 @@ Run the command:
 docker build -t caicloud/ciao .
 ```
 
-### Install Dockerized kernel
+### Install [nb2kg](https://github.com/jupyter/nb2kg)
+
+Jupyter Notebook Extension to Kernel Gateway (nb2kg) is used to communicate with the kernel in the docker image, please install the package using the command:
+
+```
+pip install nb2kg
+```
+
+Then enable it:
+
+```
+jupyter serverextension enable --py nb2kg --sys-prefix
+```
+
+### Run the kernel
 
 Run the command:
 
 ```
-./hack/install-dockerized.sh
+docker run -v {kubeconfig}:{kubeconfig} -p 8889:8889 caicloud/ciao
```

Comment:
since we are kubernetes based, why not running the remote kernel in kubernetes cluster? and thus dockerized kernel becomes kuberized kernel?

Reply:
I'm fine with this now.  I'll walkthrough the guide once #43 is ready :)

Reply:
NB. to correct my initial request, I was thinking that we should also run gateway in kubernetes.

Reply:
Yeah I am investigating https://github.com/jupyter/enterprise_gateway to see if we could run the gatway and kernel separately. Now we run gateway and kernel in one docker container.